templates/test/infopanel.html.ep

```diff
@@ -81,7 +81,10 @@
                 % }
                 % if ($worker) {
                     <div>
-                        Assigned worker: <span title="VNC port: <%= 90 + $worker->instance %>"><%= $worker->to_string %></span>
+                        Assigned worker:
+                        <span title="VNC port: <%= 90 + $worker->instance %>">
+                            %= link_to $worker->to_string => url_for('admin_worker_show', worker_id => $worker->id)
```

Comment:
Apparently this is (still) only shown if the job is running, at least this is what my quick test shows.

Also, I checked this link when I am not logged in. I click on the link, it will redirect me to the same job page again but I am logged in, then clicking again shows admin page. Weird, please crosscheck.

Also, what if the user is !operator?